configure.sh

```diff
@@ -17,57 +17,101 @@ PRIVATE_PUSHWOOSH_PROPERTIES="$BASE_PATH/android/pushwoosh.properties"
 PRIVATE_LIBNOTIFY_PROPERTIES="$BASE_PATH/android/libnotify.properties"
 PRIVATE_NETWORK_CONFIG="$BASE_PATH/android/res/xml/network_security_config.xml"
 SAVED_PRIVATE_REPO_FILE="$BASE_PATH/.private_repository_url"
+SAVED_PRIVATE_BRANCH_FILE="$BASE_PATH/.private_repository_branch"
```

Comment:
Maybe store the url/branch info together?
What if you change the repo but don't change the branch? Shall the old branch be used or the default one?

Reply:
Not an option due to backward compatibility: it will break after user checks out old version.
New configure.sh reconfigures both url and branch altogether if it is called, so "change the repo but don't change the branch" is not an issue.